assembl/graphql/extract.py

```diff
@@ -121,6 +147,8 @@ def mutate(root, args, context, info):
         require_instance_permission(CrudPermissions.DELETE, extract, context)
         for fragment in extract.text_fragment_identifiers:
             fragment.delete()
+
+        extract.tags = []
```

Comment:
Is it necessary to clear the tags before deleting the Extract ? Is the extract still stored in db if deleted ?

Reply:
Also, if the last extract that used a specific tag is deleted, what happens to the tag not used anymore ?

Reply:
yes, as with text_fragment_identifiers, we need to remove tag associations and not tags.